### PR TITLE
[Merged by Bors] - chore: fix a few typos in docstrings

### PR DESCRIPTION
**Pull request by grunweg**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Algebra/Nonarchimedean/Bases.lean [hunk 1]
```diff
@@ -327,7 +327,7 @@ theorem nonarchimedean (hB : SubmodulesBasis B) : @NonarchimedeanAddGroup M _ hB
 
 library_note "nonarchimedean non instances"/--
 The non archimedean subgroup basis lemmas cannot be instances because some instances
-(such as `MeasureTheory.AEEqFun.instAddMonoid` or `topological_add_group.to_has_continuous_add`)
+(such as `MeasureTheory.AEEqFun.instAddMonoid` or `TopologicalAddGroup.to_hasContinuousAdd`)
```
Comment (grunweg) [inline Mathlib/Topology/Algebra/Nonarchimedean/Bases.lean:330]: This comment is still wrong, I think: the API docs don't contains the new name. Help welcome.

Comment (j-loreaux) [reply Mathlib/Topology/Algebra/Nonarchimedean/Bases.lean:330]: It's just a structure projection, so it should be: 
`TopologicalAddGroup.toContinuousAdd`

Comment (grunweg) [reply Mathlib/Topology/Algebra/Nonarchimedean/Bases.lean:330]: :bulb: Thanks!
```suggestion
(such as `MeasureTheory.AEEqFun.instAddMonoid` or `TopologicalAddGroup.toContinuousAdd`)

**Comment by j-loreaux** [comment]
bors merge

**Pull request closed without merging**